services/director-v2/openapi.json

```diff
@@ -1,5 +1,5 @@
 {
-  "openapi": "3.1.0",
+  "openapi": "3.0.0",
```

Comment:
did you fix this? or is this just hard-coded?

Reply:
Nope, this should not have been commited. Thanks for pointing it out. I will revert this